### PR TITLE
Remove references to Google CLA

### PR DESCRIPTION
**Pull request by foxish**
This PR removes references to the Google CLA process which will no longer be relevant once we transition completely to the CNCF CLA.

TODO before merge:

- [x] Update wiki.
- [x] Communicate to developers via email.

---
This change is [Reviewable](https://reviewable.kubernetes.io/reviews/kubernetes/kubernetes/37028)

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -23,25 +23,18 @@ issue](https://github.com/kubernetes/kubernetes/issues/new).
...
+
   * To contribute as an individual or as am employee of a signed organization,
     [go here](https://identity.linuxfoundation.org/projects/cncf).
   * To sign up as an organization, [go
     here](https://identity.linuxfoundation.org/node/285/organization-signup).
 
-Once you are CLA'ed, we'll be able to accept your pull requests.
+Once you are CLA'ed, we'll be able to accept your pull requests. 
+For any issues that you face during this process, please refer to 
+the [CLA FAQ](https://github.com/kubernetes/kubernetes/wiki/CLA-FAQ).
```
Comment (bgrant0607) [inline CONTRIBUTING.md]: We should move the CLA FAQ to the community repo.

Comment (foxish) [inline CONTRIBUTING.md]: @sarahnovotny, where should the CLA FAQ should be moved within the community repo?

**Comment by bgrant0607** [comment]
@foxish hack/verify-munge-docs.sh failed

File: CONTRIBUTING.md (hunk 1 above)
Comment (bgrant0607) [inline CONTRIBUTING.md]: @foxish The top-level directory is fine for now. We can move it once the rest of the contributor docs are moved, if we like.

**Pull request closed without merging**